cmd/ctr/checkpoint.go

```diff
@@ -48,7 +52,32 @@ var taskCheckpointCommand = cli.Command{
 		if err != nil {
 			return err
 		}
-		fmt.Println(checkpoint.Digest.String())
+
+		crImg, err := container.Image(ctx)
+		if err != nil {
+			return err
+		}
+		labels := make(map[string]string)
+		labels["checkpoint"] = "true"
```

Comment:
`containerd.io/checkpoint` for consistency with `containerd.io/uncompressed` label?

Reply:
Done